### ISSUE TITLE
Remove references to Test Pilot?

### ISSUE THREAD

**Issue by pdehaan** [mozilla/FirefoxColor#676]
We probably want to remove the **testpilot-ga** bits and the footer links to Privacy and Terms and "About Test Pilot" redirect you to the Test Pilot EOL site.

```sh
$ git rev-parse --short HEAD # 8f7ff7b

$ git grep -i pilot

docs/acceptance.md:  It should specify that it is a Firefox Test Pilot experiment. (cf Send)
docs/acceptance.md:  It should prompt users who install Firefox Color from Firefox Color to check out Test Pilot.
docs/metrics.md:Metrics collections and analysis plan for the as yet unnamed theming experiment for [Firefox Test Pilot](https://testpilot.firefox.com).
docs/metrics.md:Data will be collected with Google Analytics and follow [Test Pilot standards](https://github.com/mozilla/testpilot/blob/master/docs/experiments/ga.md) for reporting.
package-lock.json:    "testpilot-ga": {
package-lock.json:      "resolved": "https://registry.npmjs.org/testpilot-ga/-/testpilot-ga-0.3.0.tgz",
package.json:    "testpilot-ga": "0.3.0",
src/lib/metrics.js:import TestPilotGA from "testpilot-ga";
src/lib/metrics.js:    analytics = new TestPilotGA({
src/web/index.html.ejs:    <meta name="twitter:site" content="@FxTestPilot" />
src/web/lib/components/AppFooter/AppFooter-test.js:  it("should render to About Test Pilot link", () => {
src/web/lib/components/AppFooter/AppFooter-test.js:    ).to.equal("https://testpilot.firefox.com/about");
src/web/lib/components/AppFooter/index.js:          href="https://testpilot.firefox.com/about"
src/web/lib/components/AppFooter/index.js:          About Test Pilot
src/web/lib/components/AppFooter/index.js:            href="https://testpilot.firefox.com/experiments/color"
src/web/lib/components/AppFooter/index.js:          href="https://twitter.com/FxTestPilot"
src/web/lib/components/AppFooter/index.js:          aria-label="@FxTestPilot Twitter"
src/web/lib/components/AppHeader/AppHeader-test.js:  it("should link to Firefox Test Pilot", () => {
src/web/lib/components/AppHeader/AppHeader-test.js:    ).to.equal("https://testpilot.firefox.com");
src/web/lib/components/AppHeader/index.js:              href="https://testpilot.firefox.com"
src/web/lib/components/AppHeader/index.js:              onClick={() => Metrics.linkClick("test-pilot")}
src/web/lib/components/AppHeader/index.js:              Firefox Test Pilot
src/web/lib/components/Mobile/index.js:        href="https://testpilot.firefox.com/experiments/color"
src/web/lib/components/TermsPrivacyModal/index.js:      Firefox Color is currently a Test Pilot experiment, and subject to the
src/web/lib/components/TermsPrivacyModal/index.js:      Test Pilot{" "}
src/web/lib/components/TermsPrivacyModal/index.js:      <a href="https://testpilot.firefox.com/terms">Terms of Service</a> and{" "}
src/web/lib/components/TermsPrivacyModal/index.js:      <a href="https://testpilot.firefox.com/privacy">Privacy Notice</a>. You
src/web/lib/components/TermsPrivacyModal/index.js:      <a href="https://testpilot.firefox.com/experiments/color">here</a>.
webpack.common.js:              path.resolve(__dirname, "node_modules/testpilot-ga"),
```

**Comment by caitmuenster** [comment]
Closed by PR #784. :)

**Comment by spoji** [comment]
@caitmuenster This issue is linked to a P1 still open in the PRD in the "Update Firefox Color web presence" requirement description. Should I assume this is done and should not be taken into consideration ?

**Comment by caitmuenster** [comment]
This is not quite done -- IIRC, Melanie's PR was merged but never deployed to prod. Once it's deployed, we can consider it done. 

I'm going to re-open. :/

**Comment by caitmuenster** [comment]
This should be closed with PR #826.
